example/src/androidTest/java/com/attentive/example/CreativeUITest.java

```diff
@@ -74,69 +71,91 @@ public static void classTeardown() {
 
     @Test
     public void loadCreative_clickClose_closesCreative() throws UiObjectNotFoundException {
-        loadCreative(AttentiveConfig.Mode.PRODUCTION);
-
-        // Close the Creative
-        UiObject closeButton = device.findObject(selector.resourceId("closeIconContainer"));
-        closeButton.click();
-
-        checkIfCreativeClosed();
+        synchronized (device) {
+            loadCreative(AttentiveConfig.Mode.PRODUCTION);
+
+            UiObject emailInput = device.findObject(selector.resourceId("input0input"));
+            emailInput.setText("testmail@attentivemobile.com");
+
+            // Close the Creative
+            UiObject closeButton = device.findObject(new UiSelector().resourceId("closeIconContaisdfner"));
```

Comment:
these selectors don't work on all devices, so this makes our test suite less flaky on different devices, but having multiple methods of closing